config/locales/en/devise.yml

```diff
@@ -17,7 +17,7 @@ en:
           - Sign in to view this page
         dormant: 
           - You cannot access this service
-          - This is because you have not signed in to the service for more than 6 months.
+          - This is because you have not signed in to the service for more than 90&nbsp;days.
```

Comment:
Wondering if for consistency we use `\xA0` - unless there's a reason not to that I don't know about

Reply:
Yes, annoyingly neither work when used in the other context!